coderd/notifications/notifications_test.go

```diff
@@ -869,14 +926,32 @@ func TestNotificationTemplatesCanRender(t *testing.T) {
 				Scan(&titleTmpl, &bodyTmpl)
 			require.NoError(t, err, "failed to query body template for template:", tc.id)
 
-			title, err := render.GoTemplate(titleTmpl, tc.payload, nil)
+			title, err := render.GoTemplate(titleTmpl, tc.payload, defaultHelpers())
 			require.NotContainsf(t, title, render.NoValue, "template %q is missing a label value", tc.name)
 			require.NoError(t, err, "failed to render notification title template")
 			require.NotEmpty(t, title, "title should not be empty")
 
-			body, err := render.GoTemplate(bodyTmpl, tc.payload, nil)
+			body, err := render.GoTemplate(bodyTmpl, tc.payload, defaultHelpers())
 			require.NoError(t, err, "failed to render notification body template")
 			require.NotEmpty(t, body, "body should not be empty")
+
+			partialName := strings.Join(strings.Split(t.Name(), "/")[1:], "_")
+			goldenFile := filepath.Join("testdata", "rendered-templates", partialName+"-body.md.golden")
```

Comment:
Why only the body templates?

Reply:
Are you asking about _title templates_? I can version them too.

Reply:
Correct, yeah. That would be good for completeness, since title templates can also fail to generate. 👍